### ISSUE TITLE
[android] initHybrid missing/broken in pytorch 1.4.0 nightly

### ISSUE THREAD

**Issue by phillies** [pytorch/pytorch#31419]
## 🐛 Bug

Trying to load a tensor from a blob leads to an error:
    java.lang.UnsatisfiedLinkError: No implementation found for com.facebook.jni.HybridData org.pytorch.Tensor.initHybrid() (tried Java_org_pytorch_Tensor_initHybrid and Java_org_pytorch_Tensor_initHybrid__)
        at org.pytorch.Tensor.initHybrid(Native Method)
        at org.pytorch.Tensor.initHybrid(Tensor.java:337)
        at org.pytorch.Tensor.fromBlob(Tensor.java:289)

This is a very recently introduced bug, as the nightly snapshot last week worked fine.

## To Reproduce

1. Create Android Project and add pytorch 1.4.0 nighly snapshot support (e.g. as shown here https://github.com/pytorch/pytorch/issues/29806) 
2. add following lines
        val inputHeight = 299
        val inputWidth = 299
        val inputTensorBuffer = Tensor.allocateFloatBuffer(3 * inputWidth * inputHeight)
        val inputTensor = Tensor.fromBlob( inputTensorBuffer, longArrayOf( 1, 3, inputHeight.toLong(), inputWidth.toLong()))
3. build & run app
4. App crashes with error above

## Expected behavior

Loading a tensor into memory without error (worked with nightly build last week)

## Environment

 - PyTorch Version (e.g., 1.0): 1.4.0 nightly snapshot (18 Dec 2019)
 - OS (e.g., Linux): Windows 10
 - Android studio 3.5.3, OpenJDK JRE 1.8.0_202
 - CUDA/cuDNN version: None (CPU-only)
 - GPU models and configuration: None 

## Additional context

Using a stable version is not an option, as 1.3.0 has a breaking bug in loading the inception v3 model (https://github.com/pytorch/pytorch/issues/29806)

**Comment by phillies** [comment]
Complete repo to reproduce the issue: https://github.com/phillies/pytorch_debug_android

**Comment by IvanKobzarev** [comment]
@phillies 
Thanks for reporting it.
The problem happens as native libraries at the moment are loaded only during `org.pytorch.Module.load`, as a hotfix to unblock you while we are working on the fix - if you move `Tensor.fromBlob` after `Module.load` that error should not happen

I am working on the fix.

**Comment by IvanKobzarev** [comment]
@phillies ,
The fix was merged into master as https://github.com/pytorch/pytorch/commit/3a19980b78b6638235edef367784ecc3dc37e364
I have republished android nightlies, the error should not happen if you update to the [latest nightlies](https://oss.sonatype.org/service/local/repositories/snapshots/content/org/pytorch/pytorch_android/1.4.0-SNAPSHOT/pytorch_android-1.4.0-20191220.225919-78.aar) (gradle key `--refresh-dependencies`)

**Comment by phillies** [comment]
Thanks @IvanKobzarev this error is fixed, however, now I get a segfault when I call Module.load().
`A/libc: Fatal signal 11 (SIGSEGV), code 1 (SEGV_MAPERR), fault addr 0x0 in tid 9362 (es.pytorchdebug), pid 9362 (es.pytorchdebug)`
I guess this is not related to this issue?
